content/app/development/logic/dynamic/_index.en.md

```diff
@@ -249,6 +249,36 @@ Et eksempel på en regel som er satt opp for repeterende grupper vises under:
 }
 ```
 
+### Nøstede repeterende grupper
+
+Det er også mulig å sette opp dynamikk for nøstede repeterene grupper. Oppsettet likner på repeterende grupper, men man har med et ektra parameter `childGroupId` i `repeatingGroup`-objektet, samt en ekstra indeks-indikator.
+
```

Comment:
Why is this not in english? 😅

Reply:
Just laziness and not wanting to translate a whole page to english if we are focusing on norwegian translations for the time being: https://altinndevops.slack.com/archives/C9E57MDF1/p1648816568814349

Reply:
@nkylstad English translations done ✅